src/settings/StaffSlips/components/ViewSections/StaffSlipAboutSection/StaffSlipAboutSection.js

```diff
@@ -0,0 +1,54 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { FormattedMessage } from 'react-intl';
+
+import {
+  Col,
+  Row,
+  KeyValue,
+} from '@folio/stripes/components';
+
+const StaffSlipAboutSection = ({ staffSlip }) => {
+  return (
+    <div
+      data-test-staff-slip-detail-about-section
+      data-testid="StaffSlipAboutSectionTestId"
```

Comment:
Please follow naming convention approaches
```suggestion
      data-testid="staffSlipAboutSectionTestId"
```